src/Camille.RiotGames/src/Util/RegionalRequester.cs

```diff
@@ -37,13 +35,40 @@ public class RegionalRequester
         /// </summary>
         private readonly HttpClient _client = new HttpClient();
 
+        /// <summary>
+        ///  If true, the region will be appended to the route instead of used as a subdomain.
+        /// </summary>
+        private bool _regionAsSubdomain;
```

Comment:
Make this a configuration setting (in configspec) instead of inferred

Reply:
Moved to `Camille.RiotGames.apiCallRegionConfig`, which uses a `Camille.RiotGames.RegionConfig` enum value.